department-of-veterans-affairs/caseflow
# File upload component

Connects #5819 

### Description
This PR adds a FileUpload component and adds the component to the buildScheduleUpload page.

### Testing Plan
1. Go to hearings/schedule/build/upload
1. Choose either 'RO/CO hearings' or 'Judge non-availability'
1. Upload a file
1. Confirm the redux store is updated with the filename
1. Confirm the UI displays the filename
